Cargo.toml

```diff
@@ -57,6 +59,8 @@ totp-lite = "2.0.0"
 url = "2.3.1"
 uuid = { version = "1.3.0", features = ["v4"] }
 zeroize = "1.5.7"
+rmpv = "1.0.0"
+tokio-tungstenite = { version = "*", features = ["native-tls"] }
```

Comment:
i'm trying to avoid pulling in openssl, because it makes it very hard to build for other targets (i'm trying to keep musl support) - can you use rustls instead? (you can check this using `cargo deny check`)

also, this should be an actual version specification, not `"*"`

Reply:
I updated this to use the latest version (0.18.0) and to use rustls. Funnily enough installing cargo-deny fails for me with an openssl error though, which I haven't debugged yet.